lesson07/assignment/vue-project/src/components/Home.vue

```diff
@@ -0,0 +1,82 @@
+<template>
+  <!-- Jumbotron -->
+  <b-jumbotron
+    class="rounded-0 text-light text-center d-flex justify-content-center align-items-center"
+  >
+    <!-- bg image overlay div  -->
+    <div class="img-overlay"></div>
+    <b-container class="p-0">
+      <h1 class="display-3">{{ title }}</h1>
+      <p class="lead">{{ subtitle }}</p>
+      <b-alert
+        :show="dismissCountDown"
+        dismissible
+        variant="danger"
+        @dismissed="dismissCountDown=0"
+        @dismiss-count-down="countDownChanged"
+        class="mb-3"
+      >
+        <b-container class="justify-content-md-center">
+          <!-- <b-col md="8"> -->
+          {{ message }}
+          <b-progress variant="danger" :max="dismissSecs" :value="dismissCountDown"></b-progress>
+          <!-- </b-col> -->
+        </b-container>
+      </b-alert>
+      <!-- v-if show danger button  -->
+      <b-button
```

Comment:
Great implementation of an if/else block here, also like how you added in a countdown